### ISSUE TITLE
Cancel Scrubbing

### ISSUE THREAD

**Issue by akano12** [libre-tube/LibreTube#5691]
### Describe your suggested feature

Make it possible to cancel scrubbing by swiping down the seekbar. 

### Other details

A lot of times and when watching something, i hold the scrubber and move it forward -without letting my finger off- to see if the video contains what I'm looking for. It would be great if it's possible to cancel scrubbing and let the video continue as is without having to drag the scrubber to match it with currently changing video timestamp. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Thanks to ExoPlayer, this change costed me an hour of reverse engineering their code, because they just randomly add callbacks to the player that can't be removed, so that implementing our own functionality is nearly impossible for the condition of whether the player should seek to the scrubbed position or not.

TL;DR: We now use a hacky solution that's anything but clean because ExoPlayer's API is just garbage, but it works now...

**Comment by Bnyro** [comment]
It's still debatable if the current behavior is better than your suggestion, I've just implemented it in a way that I personally feel most comfortable with (scrubbing and releasing it at the top of the screen). I'm open for other suggestions though, I just thought that swiping down can be triggered by accident too easily.
